src/main/io/gps.c

```diff
@@ -482,6 +496,41 @@ void gpsInitUblox(void)
                 }
             }
 
+            if (gpsData.messageState == GPS_MESSAGE_STATE_REFRESH_RATE) {
+                switch (gpsData.ackState) {
+                    case UBLOX_ACK_IDLE:
+                        {
+                            ubx_message tx_buffer;
+                            tx_buffer.header.preamble1 = 0xB5;
+                            tx_buffer.header.preamble2 = 0x62;
+                            tx_buffer.header.msg_class = 0x06;
+                            tx_buffer.header.msg_id = 0x08;
+                            tx_buffer.header.length = 6;
+
+                            tx_buffer.payload.cfgRate.measRate = 1000 / gpsConfig()->gps_ublox_refresh_rate_hz;
+                            tx_buffer.payload.cfgRate.navRate = 1;
+                            tx_buffer.payload.cfgRate.timeRef = 1;
+
+                            ubloxSendConfigMessage((const uint8_t *) &tx_buffer, sizeof(tx_buffer.header) + sizeof(tx_buffer.payload.cfgRate));
```

Comment:
Not sure about these sizeof(tx_buffer.header) + sizeof(tx_buffer.payload.cfgRate). Can someone take a look?

Reply:
I'd probably use `sizeof(tx_buffer)` since it's the buffer that you want to send.